### PR TITLE
fix: Pre-requisites not appearing for courses in Explore Catalog

### PR DESCRIPTION
**Pull request by johnnagro**
## Description

- added the correct field for indexing
- added a test to check retrieved vs indexed fields

## References

- [ENT-5415](https://openedx.atlassian.net/browse/ENT-5415)

### CODE REVIEW

File: enterprise_catalog/apps/catalog/algolia_utils.py [hunk 1]
```diff
@@ -41,6 +41,7 @@
     'partner',
     'partners',
     'prerequisites',
+    'prerequisites_raw',
```
Comment (johnnagro) [inline enterprise_catalog/apps/catalog/algolia_utils.py:44]: we were missing this field in the indexing

File: enterprise_catalog/apps/api/v1/tests/test_export_utils.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+from django.test import TestCase
+
+from enterprise_catalog.apps.api.v1 import export_utils
+from enterprise_catalog.apps.catalog import algolia_utils
+
+
+class ExportUtilsTests(TestCase):
```
Comment (johnnagro) [inline enterprise_catalog/apps/api/v1/tests/test_export_utils.py:7]: this will fail if we ever try to add a field for export which we're not indexing

Comment (alex-sheehan-edx) [reply enterprise_catalog/apps/api/v1/tests/test_export_utils.py:7]: great test

**Review by alex-sheehan-edx — APPROVED** [review]
lgtm

**Pull request merged**